docs/developer/release/build.md

```diff
@@ -1,18 +1,16 @@
 # Build release artifacts
 
 Follow the sections below, depending on the OS you want to build artifacts for.
-Note that you can build Linux artifacts from macOS, since we use containers to
-do so.
+Linux artifacts can be built from macOS, via the use of containers.
```

Comment:
```suggestion
Linux artifacts can be built on macOS using containers.
```

Reply:
Thanks. I applied one half of your suggestion, as I think [the "containers" project from macOS](https://github.com/apple/container) might confuse some people otherwise.